### PR TITLE
Fix other issue in portuguese translation

### PR DESCRIPTION
**Pull request by qgustavor**
There was a lot of mistakes in this translation!

**Edit:** ~~I'll review a little more.~~ done.

### CODE REVIEW

**Comment by Pizzacus** [comment]
Thanks for the fix! Didn't expect to see a "veteran translator" back on the repo. 👀

Ignore the ZEIT Now deploy I was messing with stuff on the repo

File: src/locales/pt.json [hunk 1]
```diff
@@ -110,7 +110,7 @@
 	},
 	"the-others-are-trash": {
 		"large-text": "Os outros personagens são um lixo, sem exceções.",
-		"sub-text": "Elas podem ser bonitinhas, mas quando você sabe os fatos, na realidade elas não são.",
+		"sub-text": "Elas podem parecer bonitinhas, mas quando você descobre os fatos, percebe que elas não são." então,
```
Comment (qgustavor) [inline src/locales/pt.json:113]: There is a mistake here, SORRY.

Comment (qgustavor) [reply src/locales/pt.json:113]: The fix:

```diff
- "sub-text": "Elas podem parecer bonitinhas, mas quando você descobre os fatos, percebe que elas não são." então,
+ "sub-text": "Elas podem parecer bonitinhas, mas quando você descobre os fatos, percebe que elas não são.",
```

**Comment by Pizzacus** [comment]
Oh... I forgot the deploy doesn't check if the JSON is valid, I should have actually verified the preview, I am sorry too. :x

**Comment by qgustavor** [comment]
I copy pasted without thinking. I was sharing the website with some friends
and I quickly realized I translated wrongly a cognate, then a friend of
mine found that last mistake. It's a bit frustrating finding those things
after so much time.

2020年4月2日(木) 20:33 Pizzacus™ <notifications@github.com>:

**Comment by Pizzacus** [comment]
To err is human, to edit, divine. ❤️ 

To be fair if you think about it, the satania.moe website *itself* is pretty low quality. I had poor writing skills back then, and it shows, and I assume this translates poorly. And frankly, the design, programming, and even humour of the website is pretty bodged. So really, I don't mind the translation mistakes at all. xD

Interestingly, we get a LOT more visits from Brazil than Portugal. And it has been this way for as long as I can remember, I am unsure why. But you would think someone would wanna contribute a version adapted for Brazilian Portuguese, but no. But then I am not sure if Brazilian users use the Portuguese version or the English version. Unfortunately, I didn't collect analytics on that :c.

Also, I do know the Korean version is apparently brilliant. All the Korean people I've asked to check said they preferred it over the English version. Apparently, the translator was able to use Korean internet slangs and cleverly localise some jokes to create a greater comedic effect.

This is great because we get a surprising amount of traffic from South Korea, as satania.moe is mentioned on namu.wiki, a Korean wiki that is a bit of a blend between Wikipedia and TVtropes, it is incredibly popular (19th most visited in South Korea when Wikipedia is 14th)

image


But eh, that's still not that many people. The website used to get 3k visits a month in 2018, now it only gets 1k. It is such a shame I didn't track visits on release day, because I know it got a ton of traffic from the number of people that got into our Disord, but I have no idea how much :c

**Comment by qgustavor** [comment]
I'm Brazilian. Well, Wikipedia don't have separate editions for different
portuguese variants, so I think it works for both Portuguese and Brazilian
people.

2020年4月2日(木) 21:20 Pizzacus™ <notifications@github.com>:

**Comment by Pizzacus** [comment]
<sup>Wait seriously?! I always thought the Portuguese satania.moe was written in the Portuguese dialect... No idea why I made this assumption tho.</sup>

Wikipedia's case is interesting. They tend not to separate languages because it would segregate users of their community. If they had different versions of the Portuguese Wikipedia, they would each have less contributors, and thus be of lower quality.

The same train of thought was used for the Mandarin (Chinese) version. Except Chinese is significantly more challenging.

Chinese has an alphabet of thousands of characters, each representing somewhat of a concept. That shouldn't be too unfamiliar to you since you're from the anime fandom as well xD. However, Chinese can be written in either the traditional script, which exists since centuries, or the simplified script, which is a reform from the Chinese Communist Party. (Japanese mixes the two, so there are actually 4 scripts used in Japanese 😨)

This created two variants of Chinese, Simplified Chinese and Traditional Chinese. They are both Chinese in the end, they are mutually intelligible, but not mutually readable. Most of greater China speaks, or at least write in Chinese. But while the mainland uses simplified, Hong Kong and Taiwan uses Traditional.

Worse, Hong Kong's Chinese differs from Taiwan's, as they use a lot more loan words because of their British influence. These create more confusion in the Chinese language than in Latin languages.

Because Wikipedia didn't want to make distinct communities for what is essentially the same language, in the end, they created a conversion system, where pages get converted to the script of your choice. It is much harder than simple character substitution and still requires manual tooling every now and then on a page-by-page basis. This shows you the lengths Wikipedia will go to avoid splitting people up.

And I think Wikipedia is right. It is pretty useless to distinguish dialects when your goal is just to share information with foreign speakers. Because they *need* this information, they won't care how it is presented to them.

However, when your goal is to create engagement, to create an app that feels like home when you use it on the daily, or to entertain people, it is important to speak your user's language fluently. It's all about Internationalisation vs. Localisation in the end.

And satania.moe falls into the latter category. Because it is a comedic website, because it aims to entertain, it makes sense to strive to adapt the text to fit the audience. People will enjoy the content a lot more if it speaks to them directly, if it feels like something they would write.

And this is why greatly appreciate translators who took the courage to change the text and rewrite entire jokes that's how you make use of the strengths of your language to improve over the English version.

But yeh so that's why we *might* benefit from both versions, if there is content to change.

**Comment by qgustavor** [comment]
Well, unlike chinese, the differences are are basically just some words and
expressions. I sometimes watch anime in portuguese from Portugal without
any issue, but I know some people that don't understand some words and
avoid those translations. Maybe in future someone will make a Portuguese
translation of satania.moe.

2020年4月2日(木) 22:28 Pizzacus™ <notifications@github.com>:

**Comment by Pizzacus** [comment]
Really? That's interesting 🤔 

I always thought Portuguese and Brazilian were different enough that you could tell which one it is at a first glance, but maybe they are more similar than I thought. Hmm...

Well if there is not enough content to change to merit a new version, it really doesn't matter I guess. I asked a Spanish friend and he said that if the website was to be rewritten into Latin American Spanish only the veeeerry last title would change. But I thought Portuguese would have more significant changes...

Like idk when I talk to Portuguese-speaking friends about this, they're always claiming the two are super different, but maybe that's a bit of an exaggeration...

**Pull request merged**